src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -764,162 +982,12 @@ def launch_fluent(
             )
 
     new_session, meshing_mode, argvals, mode = _get_session_info(argvals, mode)
+    argvals = locals().copy()
```

Comment:
I think line 962, 986, 987 are all redundant with this line.

Reply:
@mkundu1 

Yes, I agree. We don't need line 986 and 987 now after adding `argvals = locals().copy()`

We need line 962 because we are using it at line 984 `_get_session_info(argvals, mode)` otherwise we get following error.

![image](https://github.com/ansys/pyfluent/assets/106588300/fe5f1376-a6de-4ecb-8110-da026aa19a7b)

Reply:
Done. Thank you.

Reply:
We need line 986 i.e. `argvals["argvals"] = argvals` otherwise we get following error.

![image](https://github.com/ansys/pyfluent/assets/106588300/5e0e9427-d828-4eb0-83ef-ad368848e220)